### ISSUE TITLE
Adding source ip:port failed

### ISSUE THREAD

**Issue by misaalanshori** [AhMyth/AhMyth-Android-RAT#153]
![Screenshot_20190920-103924_1](https://user-images.githubusercontent.com/38964181/65298509-fa739680-db95-11e9-9b32-e51e3998eb89.jpg)

Did i miss a step? Build apk failed with "Adding source ip:port failed", i have inserted my server public ip and the default ip in to the textbox and clicked build

**Comment by duhjja** [comment]
Try private IP if that does not work you may be behind a dynamic ip address and will need a static ip

**Comment by misaalanshori** [comment]
Its on a vps with a static ip which is why i blurred it. But the ip probably wont make the apk building proccess fail

**Comment by duhjja** [comment]
Try using a different source port not default ie. 4444.

**Comment by misaalanshori** [comment]
Source ip and port is the ip and port of the server right?

**Comment by duhjja** [comment]
Yes it is your ip and a port you choose as long as not otherwise used by something else ie vpn is like 13750 as example not actual. by looks of the screen grab you provided it seems like your port is being rejected since it looks like you are building in a linux environment you may actually be able to do the exact thing in windows and when built use linux to listen to the port after apk is deployed.

**Comment by misaalanshori** [comment]
So what youre saying here is that i should try building the apk on windows and then point it to my server ip?

**Comment by duhjja** [comment]
I did that exact thing when I first started using this. I attempted to run the full program and .apk build in kali and ran into many issues so I built on windows. I still run into hiccups running in kali but have to build in windows then can run app.
